Tasks/VsTest/vstest.ts

```diff
@@ -24,7 +23,7 @@ try {
     var vsTestVersion: string = tl.getInput('vsTestVersion');
     var vstestLocationMethod: string = tl.getInput('vstestLocationMethod');
     var vstestLocation: string = tl.getPathInput('vsTestLocation');
-    var testAssembly: string = tl.getInput('testAssembly', true);
+    var testAssembly: string[] = tl.getDelimitedInput('testAssembly', '\n', true);
```

Comment:
\n is being used in other tasks as well. this change is required to have uniformity in input between all tasks.